examples/05_glm_second_level/README.txt

```diff
@@ -1,5 +1,6 @@
-GLM : Second level analysis examples
--------------------------------------
+===========================
+GLM : Second level analysis
```

Comment:
I think we need to remove the space before the colon, here, to standardize with `GLM: First Level Analysis` !

```suggestion
GLM: Second level analysis
```